django_eth_events/chainevents.py

```diff
@@ -1,25 +1,41 @@
 from abc import ABCMeta, abstractmethod
+from typing import Optional, List
 
 from .singleton import SingletonABCMeta
 
 
-class AbstractAddressesGetter(object):
+class AbstractAddressesGetter(metaclass=SingletonABCMeta):
     """Abstract AddressesGetter class."""
-    __metaclass__ = SingletonABCMeta
 
     @abstractmethod
-    def get_addresses(self): pass
+    def get_addresses(self) -> List: pass
 
     @abstractmethod
-    def __contains__(self, address): pass
+    def __contains__(self, address: str) -> bool: pass
 
 
-class AbstractEventReceiver(object):
+class AbstractEventReceiver(metaclass=ABCMeta):
```

Comment:
I would do `class AbstractEventReceiver(ABC):`, but it doesn't matter, it's the same: https://docs.python.org/3/library/abc.html#abc.ABC

Reply:
Yes, I would get rid of ABCMeta as well, I just migrated from the previous `__metaclass__` definition to the newly one.